src/hipscat/io/paths.py

```diff
@@ -0,0 +1,35 @@
+"""Methods for creating partitioned data paths"""
+
+import os
+
+
+def pixel_directory(catalog_path="", pixel_order=0, pixel_number=0):
+    """Create path *string* for a pixel directory. This will not create the directory.
+
+    Args:
+        catalog_path (str): base directory of the catalog (includes catalog name)
+        pixel_order (int): the healpix order of the pixel
+        pixel_number (int): the healpix pixel
+    Returns:
+        string of the form "<catalog_path>/Norder<pixel_order>/Npix<pixel_number>/"
+    """
+    return os.path.join(
+        catalog_path, f"Norder{int(pixel_order)}/Npix{int(pixel_number)}"
+    )
+
+
+def pixel_catalog_file(catalog_path="", pixel_order=0, pixel_number=0):
+    """Create path *string* for a pixel catalog file. This will not create the directory or file.
+
+    Args:
+        catalog_path (str): base directory of the catalog (includes catalog name)
+        pixel_order (int): the healpix order of the pixel
+        pixel_number (int): the healpix pixel
```

Comment:
Do these render properly with sphinx? Usually there needs to be spaces around the `:` if it's numpydoc

Reply:
Probably not.

That feels like a big undertaking, and you're right that CI and docs are going to be added in separate PRs. I've created issue/3 to track fixes to documentation for initial push to autodoc.